app/models/library_entry.rb

```diff
@@ -193,9 +193,21 @@ def kind
     media.trending_vote(user, 1.0) if status_changed?
   end
 
-  after_commit ->() { sync_entry(:create) }, on: :create
-  after_commit ->() { sync_entry(:update) }, on: :update
-  after_destroy ->() { sync_entry(:delete) }
+  after_commit(on: :create) do
+    sync_entry(:create) # mal exporter
+    # Stat STI
+    Stat::AnimeGenreBreakdown.increment(user, media.genres)
```

Comment:
heh, well I know what is wrong but my _easy_ solution of just doing a `before do subject.recalculate!` fails it seems...